src/rebar_git_resource.erl

```diff
@@ -11,6 +11,9 @@
 
 -include("rebar.hrl").
 
+%% Regex used for parsing scp style remote url
+-define(SCP_PATTERN, "\\A(?<username>[^@]+)@(?<host>[^:]+):(?<path>.*)(\\.git)?\\z").
```

Comment:
Unless I'm wrong, you'll want `\\\\.git` to end the pattern, otherwise the regex will interpret it as `\.` meaning a `.` -- same with the other escape sequences:

```
re:run("user@host:patha/pathbgit", "\\A(?<username>[^@]+)@(?<host>[^:]+):(?<path>.*)(\\.git)?\\z", [{capture, [host, path], list}]).
{match,["host","patha/pathbgit"]}
```
Do note also that just `.git` is not a valid path, so probably we'll want `<path>` to refer to `.+`.